light-rest-4j/src/main/java/com/networknt/codegen/rest/OpenApiGenerator.java

```diff
@@ -273,48 +286,123 @@ public void generate(String targetPath, Object model, Any config) throws IOExcep
                     if (type == null) {
                         throw new RuntimeException("Cannot find the type of \"" + modelFileName + "\" in #/components/schemas/ of the specification file.");
                     }
-                    if (!"object".equals(type)) {
-                        continue;
-                    }
 
-                    if(!overwriteModel && checkExist(targetPath, ("src.main.java." + modelPackage).replace(".", separator), modelFileName + ".java")) {
-                        continue;
-                    }
-                    if (isEnumClass) {
-                        transfer(targetPath, ("src.main.java." + modelPackage).replace(".", separator), modelFileName + ".java", templates.rest.enumClass.template(modelPackage, modelFileName, enums));
-                        continue;
+                    if ("object".equals(type)) {
+                        if (!overwriteModel && checkExist(targetPath, ("src.main.java." + modelPackage).replace(".", separator), modelFileName + ".java")) {
+                            continue;
+                        }
+
+                        Runnable r;
+                        if (isEnumClass) {
+                            final String e = enums;
+                            r = () -> {
+                                try {
+                                    transfer(targetPath,
+                                            ("src.main.java." + modelPackage).replace(".", separator),
+                                            modelFileName + ".java",
+                                            templates.rest.enumClass.template(modelPackage, modelFileName, e));
+                                } catch (IOException ex) {
+                                    throw new RuntimeException(ex);
+                                }
+                            };
+                        } else {
+                            r = () -> {
+                                final int referencesCount = references.size();
```

Comment:
If this line is only for testing, can it be commented out?

Reply:
It's not for testing... looks like while refactoring I missed out some code. I'll either add that code or remove this line.

Reply:
This one is addressed.